redaxo/src/addons/backup/boot.php

```diff
@@ -1,5 +1,9 @@
 <?php
 
-if (rex_addon::get('cronjob')->isAvailable() && !rex::isSafeMode()) {
-    rex_cronjob_manager::registerType(rex_cronjob_export::class);
+if (!rex_addon::get('cronjob')->isAvailable()) {
+    return;
 }
+if (rex::isSafeMode()) {
+    return;
+}
+rex_cronjob_manager::registerType(rex_cronjob_export::class);
```

Comment:
Das ist die boot.php des Backup-Addons.
Bisher enthält die eine Anweisung, und zwar wenn das Cronjob-Addon auch aktiv ist, dann wird dazu der Typ registriert.
Hier finde ich das Early-Return-Konzept fehl am Platz. 
Denn wenn man irgendwas ergänzen wird in dieser Datei, wird es höchstwahrscheinlich nicht mehr bzgl cronjob-Addon sein, und die Early-Returns passen nicht mehr. Dann muss man es wieder umstellen.